### PR TITLE
31083 list links styling

### PR DESCRIPTION
**Pull request by bienvu**
The list of links in many blocks differs from the standard ul styling. 
This styling is the same as used in the sidebar navigation.

https://redmine.previousnext.com.au/issues/31083

### CODE REVIEW

File: themes/agov/agov_whitlam/sass/components/list-links/_list-links.scss [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    padding: 0;
+    margin-bottom: 9px;
+    list-style: none;
+
+    &.is-expanded {
+      list-style: none;
+
+      ul {
+        padding-left: 28px;
+      }
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/list-links/_list-links.scss]: This `ul` ruleset needs to be nested inside .list-links, but it doesn't need to be nested this deep; its selector is more complex than it should be: `.list-links li.is-expanded ul`. `.list-links ul` is sufficiently specific.

File: themes/agov/agov_whitlam/sass/components/list-links/_list-links.scss [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    }
+
+    &.is-collapsed {
+      list-style: none;
```
Comment (JohnAlbin) [inline themes/agov/agov_whitlam/sass/components/list-links/_list-links.scss]: `list-style: none` is added 3 times now. Remove this one and the one in `is-expanded`. The one in `li` is sufficient, isn't it?

**Comment by bienvu** [comment]
Fixed!
Please check.

**Comment by kimpepper** [comment]
Feedback has been resolved.

**Pull request merged**